bin/direxec

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+
+import os
+import subprocess
+
+mode = os.path.basename(__file__)
+
+dir_odoo = '/opt/odoo'
+dir_common = os.path.join(dir_odoo, 'common', '%s.d' % mode)
+dir_custom = os.path.join(dir_odoo, 'custom', '%s.d' % mode)
+
+files_common = [
+    os.path.join(dir_common, d) for d in os.listdir(dir_common)
+]
+files_custom = []
+
+if os.path.isdir(dir_custom):
+    files_custom = [
+        os.path.join(dir_custom, d) for d in os.listdir(dir_custom)
+    ]
+
+files_all = files_common + files_custom
+sorted(files_all)
+
+for command in files_all:
+    if os.access(command, os.X_OK):
+        subprocess.check_output(command, shell=True)
```

Comment:
I tried that originally & it was failing - I forget on which script now. Seems like some bash scripts require a shell? 🤷‍♂️